x-pack/test/functional/apps/infra/feature_controls/logs_security.ts

```diff
@@ -59,7 +59,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       it('shows logs navlink', async () => {
         const navLinks = (await appsMenu.readLinks()).map((link) => link.text);
-        expect(navLinks).to.eql(['Overview', 'Logs', 'Stack Management']);
+        expect(navLinks).to.eql(['Overview', 'Alerts', 'Logs', 'Stack Management']);
```

Comment:
@weltenwort If it is about the Observability menu, `Stack Management` shouldn't be here, right? This option appears in a separate menu `Management`. But maybe I am missing something here.

Reply:
This list seems to contain all leaf nodes in the Kibana navigation menu. Since the role is defined as

```typescript
{
          elasticsearch: {
            indices: [{ names: ['metricbeat-*'], privileges: ['read', 'view_index_metadata'] }],
          },
          kibana: [
            {
              feature: {
                logs: ['all'],
              },
              spaces: ['*'],
            },
          ],
}
```

in the test's `before` function, doesn't this list seems to be correct?

I agree that it's slightly confusing that the section headers are not part of the list, but that's how the test framework service seems to work.

Reply:
Thanks for clarifying!